templates/token.tmpl

```diff
@@ -0,0 +1,24 @@
+<div class="container text-center">
+  <h1>Token: <code>{{ .Token }}</code></h1>
+</div>
+<div class="container text-center top-buffer">
+  <p>Success! Your token can now be used with this authenticator by placing it the <b>users</b> section of your Kubeconfig file.</p>
+</div>
+<div class="container top-buffer">
+  <div class="panel panel-primary">
+    <div class="panel-heading">
+      <h3 class="panel-title">Sample Kubeconfig</h4>
```

Comment:
As per previous conversation, another reminder to give a hint for `kubectl config set-credentials` rather than manual copy/paste